TranslationAssistant.Business/DocumentTranslationManager.cs

```diff
@@ -650,6 +699,7 @@ private static void ProcessPowerPointDocument(string outputDocumentFullName,stri
                                         {
                                             Text = newValue
                                         };
+                                        counts.TargetCharCount += newValue.Length;
```

Comment:
Isn't `var batch = lstComments.Select(text => text.InnerText)` the corresponding SourceCharCounts here?

Reply:
I missed it!  Thanks!